### DISCUSSION TITLE
Data analysis

### DISCUSSION THREAD

**Discussion by aed0n** [obsproject/obs-studio#6734] [Development Questions]
Dear all,

I am kinda new to Github, so I do not really know how it works. In the past I have used it a lot for certain scripts, but I am no longer a programmer!

I have a specific request. For my data analysis activities I am looking to record certain live streams (not for commercial purposes) from a specific OTT platform. It has to be in at least 1080P

Who can help me?

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
